### DISCUSSION TITLE
Extension activation failed: connect ECONNREFUSED

### DISCUSSION THREAD

**Discussion by mateusdayrell** [community/community#13486] [Copilot Conversations]
After installing copilot on VS code i got this error:
GitHub Copilot could not connect to server. Extension activation failed: "connect ECONNREFUSED 20.201.28.148:443"

**Comment by davecheney** [comment]
Is it possible that your employer, internet service provider, or government may be filtering your internet connection or intercepting your HTTPS traffic?

**Reply by mateusdayrell** [reply]
Yes, it is. I am using a proxy

**Comment by mateusdayrell** [comment]
I've already made the proxy configuration on Visual Studio Code but the problem remains.

**Reply by eatingfirst-t** [reply]
Have you solved it?

**Reply by mateusdayrell** [reply]
No
